### ISSUE TITLE
Improve vega/vega-lite file extension handling

### ISSUE THREAD

**Issue by ellisonbg** [jupyterlab/jupyterlab#2858]
* Include plain `.json` as a possible file extension.
* Get rid of the `.vl.json` and `.vg.json` as they are covered by `.json`.
* Make sure it is the default opener for `.vl` and `.vg`

**Comment by blink1073** [comment]
If there are `.vl.json` files, wouldn't you want them to be opened by this renderer by default?

**Comment by ellisonbg** [comment]
Yes, ideally. However, our file extension matching is such that those files are currently matches first by the `.json` pattern so they are opened in the text editor. We would need to change the file extension manager to prioritize the longest match first when there are common endings. Until we do that I am fine removing these patterns.

**Comment by blink1073** [comment]
In master, a `.vl.json` file opens using the Vega renderer.  We prioritize a longer extension match.
